src/fuselage/CCPACSFuselageProfiles.h

```diff
@@ -40,6 +40,8 @@ class CCPACSFuselageProfiles : public generated::CPACSFuselageProfiles
     // Read CPACS fuselage profiles
     TIGL_EXPORT void ReadCPACS(const TixiDocumentHandle& tixiHandle, const std::string& xpath) OVERRIDE;
 
+    TIGL_EXPORT virtual CCPACSFuselageProfile& AddFuselageProfile() OVERRIDE;
```

Comment:
Could you remove virtual? As far as I understood, override implies a virtual function.

I used the clang-tidy tool to add override to all missing methods. This tool doesn't use virtual, whenever override is used.

Reply:
This is correct, override implies virtual. I'm just used to having virtual before every virtual member function.

Great to see you use clang-tidy! I have not used it before, but i heard great things!